### PR TITLE
CB-19023: Fix failure reason display in events generated by backup failures.

### PR DESCRIPTION
**Pull request by sxxgrc**
Jira: https://jira.cloudera.com/browse/CB-19023

Simply added a new method to the status service for accepting a message argument for the event sent with the other parameters.

### CODE REVIEW

**Review by yg-gd — APPROVED** [review]

**Review by RokolyA94 — APPROVED** [review]

**Review by lajosrodek — APPROVED** [review]
LGTM

**Comment by lajosrodek** [comment]
Waiting for Raven approval

**Review by sodre90 — APPROVED** [review]

**Pull request merged**
